### DISCUSSION TITLE
Partial Document Update GA

### DISCUSSION THREAD

**Discussion by kipusoep** [Azure/azure-cosmos-dotnet-v3#2856] [Q&A] [answered]
I read the blog [about the GA of Partial Document Update](https://devblogs.microsoft.com/cosmosdb/partial-document-update-ga/) and it states:

> Partial document update is currently supported in Azure Cosmos DB **.NET SDK**, Java SDK, Node SDK, and stored procedures.

We can only find the stuff [marked as internal because of the preview flag](https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/Microsoft.Azure.Cosmos/src/Patch/PatchOperation.cs#L14). When can we expect a release in the SDK?

**Comment by j82w** [comment, marked as answer]
We are currently working on a new SDK release to move the APIs to the GA SDK. We are hoping to have a release by the end of next week (November 12th). 

The APIs should not change so you can use the preview SDK until the new SDK is released. You should only have to change the nuget version once it is released.

**Reply by j82w** [reply]
3.23.0 contains the new patch APIs. https://github.com/Azure/azure-cosmos-dotnet-v3/releases/tag/3.23.0
